### PR TITLE
Allow setting environment variables for all pods

### PR DESCRIPTION
**Pull request by oasisvali**
Since TrinoDB allows Environment Variables to be referenced in configuration files and recommends this approach for injecting secrets (https://trino.io/docs/current/security/secrets.html), this PR allows setting common environment variables for the coordinator and worker pods by adding an `envVars` value.

The `envVars` value is an array of objects in the Kubernetes Container Environment Variables format e.g.

```
envVars:
  - name: S3_SECRET_KEY
    valueFrom:
      secretKeyRef:
        name: aws-secret
        key: secret-key
  - name: S3_ENDPOINT
    value: https://s3.site.com
```

The configuration files can then access the secret via the Environment Variable

```
hive.s3.aws-access-key=${ENV:S3_ACCESS_KEY}
```

### CODE REVIEW

**Comment by martint** [comment]
@cla-bot check

**Comment by hashhar** [comment]
I think this is also possible to do when templating the configMap that gets written as the catalog config file. What is the difference here? Both of them look same to me from a usability perspective.

cc: @przemekak @aalbu

**Comment by oasisvali** [comment]
@hashhar I'm not sure it's possible to inject secret values into the catalog config. k8s only allows injecting secret values as Environment Variables, not in configmaps afaik. For sensitive data such as s3 access & secret keys it would be preferred to store inside a secret and then inject it as an environment variable.

**Comment by oasisvali** [comment]
Any updates on this @martint @przemekak @aalbu?

I feel like allowing to specify environment variables is the cleanest and most k8s-conformant way to pass secret values into the config.

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ The following table lists the configurable parameters of the Trino chart and the
 | `additionalConfigProperties` |  | `{}` |
 | `additionalLogProperties` |  | `{}` |
 | `additionalCatalogs` |  | `{}` |
+| `envVars` | Common Environment Variables applied to all Coordinator and Worker Pods | `[]` |
```
Comment (nineinchnick) [inline README.md:36]: How about naming it just `env` to match the Pod spec? We could also add `envFrom` but can do that in another PR.

https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#container-v1-core

File: charts/trino/Chart.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.0
+version: 0.2.1
```
Comment (nineinchnick) [inline charts/trino/Chart.yaml:18]: I'd raise it to `0.3.0` since adding support for environment variables sounds like a new feature.
